root/frontend/src/constants/CardColumn.ts

```diff
@@ -1,12 +1,21 @@
+const vertGap = 10;
+
+function calc(sizeList: number) {
```

Comment:
Calculations of any regard shouldn't exist in constant files. I would say the same applies for calculating `screenHeight` below.